client/my-sites/stats/activity-log/index.jsx

```diff
@@ -381,8 +381,8 @@ class ActivityLog extends Component {
 				{ this.props.siteIsOnFreePlan && <UpgradeBanner siteId={ siteId } /> }
 				{ config.isEnabled( 'rewind-alerts' ) && siteId && <RewindAlerts siteId={ siteId } /> }
 				{ siteId &&
-					'unavailable' === rewindState.state && <UnavailabilityNotice siteId={ siteId } /> }
-				{ 'awaitingCredentials' === rewindState.state && (
+				'unavailable' === rewindState.state && <UnavailabilityNotice siteId={ siteId } /> }
+				{ 'awaitingCredentials' === rewindState.state && ! this.props.siteIsOnFreePlan && (
 					<Banner
```

Comment:
Looks pretty good and thanks for contributing! Two minor nit-picks:

- the line starting with `unavailable` should be have an additional indentation
- instead of `this.props.siteIsOnFreePlan` you can simply write `siteisOnFreePlan` because the simpler version is initialized above

Reply:
`siteisOnFreePlan` is not defined...

Reply:
I think you may need a re-base.

Reply:
Makes sense...

Reply:
I have re-started the branch from `master` just to make sure, but I still don't see the `siteisOnFreePlan` variable you are talking about...